lib/ExternalModule.js

```diff
@@ -302,7 +302,7 @@ class ExternalModule extends Module {
 			const exportsInfo = chunkGraph.moduleGraph.getExportsInfo(this);
 			for (const exportInfo of exportsInfo.orderedExports) {
 				hash.update(exportInfo.name);
-				hash.update(exportInfo.getUsedName() || "");
+				hash.update(/** @type {string} */ (exportInfo.getUsedName()) || "");
```

Comment:
In 3.8 `getUsedName` was returning `any` with insufficient parameters, but now returns `string | false | typeof SKIP_OVER_NAME`, which is not usable as `hash`'s key. I'm not sure this cast is safe, but I didn't want to change anything but types in this PR.